docs/modules/ROOT/examples/map.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: hazelcast.com/v1alpha1
+kind: Hazelcast
+metadata:
+  name: map-sample
+spec:
+  hazelcastResourceName: hazelcast-sample
```

Comment:
What is this referring to? Do we have a more complete example that shows how to add indexes etc..?